src/io/flutter/run/bazelTest/BazelTestRunner.java

```diff
@@ -127,10 +128,10 @@ public void onTextAvailable(@NotNull ProcessEvent event, @NotNull Key outputType
           stdoutParser.appendOutput(text);
 
           for (String line : stdoutParser.getAvailableLines()) {
-            if (line.startsWith("[{")) {
+            if (line.startsWith("{")) {
```

Comment:
Is this intentional? Do we not receive leading `[{...` here?

Reply:
This is an artifact of another change I'm working on; I'll revert this.